### PR TITLE
add workflow to submit talks

### PR DESCRIPTION
**Pull request by tormath1**
Following discussions on Telegram and in the linked issue, we add a GH workflow in order to ease talk submission for the next meetups (user workflow is described in the issue).

A running example can be found on my fork in the following issue: https://github.com/tormath1/meetups/issues/6. You can ofc try it by yourself.

Some initial feedback:

- Finally, I'm not a big fan of using checkbox for the slot time / language since it will show the issue as "incomplete"
- Python tests will never work on this repo since the action listens on `main` branch and we still have a `master` branch
- A dedicated label called "proposal" needs to be created

closes https://github.com/sre-paris/meetups/issues/12

### CODE REVIEW

**Review by pabluk — COMMENTED** [review]
Great! thanks @tormath1 for this contribution :+1: 

I could maybe suggest to split this PR and move to another PR the issue template, in order to be test the render regarding checkboxes.

About the default branch name, I'm ready to follow [these instructions](https://github.com/github/renaming#renaming-existing-branches) to rename it to `main` if you it's ok for you as well (that will require updating your local copy).

**Comment by pabluk** [comment]
By the way, I've just created the [proposal](https://github.com/sre-paris/meetups/labels/proposal) label, don't hesitate to rename it if needed.

**Comment by tormath1** [comment]
@pabluk a dedicated issue for template has been created: #14. :) 

> About the default branch name, I'm ready to follow these instructions to rename it to main if you it's ok for you as well (that will require updating your local copy).

I'm totally fine with it !

**Comment by pabluk** [comment]
> > About the default branch name, I'm ready to follow these instructions to rename it to main if you it's ok for you as well (that will require updating your local copy).
> 
> I'm totally fine with it !

Perfect! I've just renamed the default branch, following Github's recommendation we need to run these command on every local copy:
```
git branch -m master main
git fetch origin
git branch -u origin/main main
```

**Comment by tormath1** [comment]
@pabluk any news on this ? :D

**Comment by pabluk** [comment]
hey @tormath1! sorry I didn't get that this PR was ready to review after the #14, I'll take a look at it asap.

File: scripts/create-content/test_main.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    def test_parse_issue(self, mock_stdout):
+        json_data = r"""
+            {
+                "issue": {
+                    "title": "First talk",
+                    "created_at": "2020-09-12T11:19:56Z",
+                    "user": {
+                        "login": "tormath1"
+                    },
+                    "body": "Some description\n"
```
Comment (pabluk) [inline scripts/create-content/test_main.py:65]: As a suggestion, maybe could be better to adapt the `body` field with content coming from the new template.

For example, taking the issue #6 and extracting the json payload from https://api.github.com/repos/tormath1/meetups/issues/6.json we have:
```json
"body": "__author name__: Mathieu Tortuyax\r\n\r\n__author bio__: My bio\r\n\r\n__expected time__ :\r\n\r\n- [x] lightning talk (~ 5 min)\r\n- [ ] 15 min\r\n\r\n__language__:\r\n\r\n- [x] :fr:\r\n- [ ] :uk:\r\n\r\n__abstract__: in this talk, we will talk about linux\r\n"
```

**Review by pabluk — APPROVED** [review]
Everything seems good :+1: I just added a suggestion but maybe could be better to keep tests as simple as possible.

So if it's ok for you, feel free to merge these changes and thank you @tormath1 for your contribution :pray:

File: scripts/create-content/test_main.py (hunk 1 above)
Comment (tormath1) [reply scripts/create-content/test_main.py]: Done. :)

**Pull request merged**